src/utils/walletIntegration/wallets/hoogiiWallet.tsx

```diff
@@ -110,6 +110,14 @@ class hoogiiWallet implements WalletIntegrationInterface {
     // Hoogii wallet balance retrieval logic
     console.log('Getting Hoogii Wallet Balance')
   }
+
+  getAddress() {
+    
+  }
+
+  async addAsset(assetId: string, symbol: string, logo: string): Promise<void> {
+    console.log('Currently only Goby has support for adding an asset programatically')
```

Comment:
toast?

Reply:
This function will never be hit as "add asset to wallet" button only shows for supported wallets. Added it here for consistency. Now that I think of it, I may as well just convert this to a toast if it somehow gets hit 👍